### DISCUSSION TITLE
Qusetion on Parameter Optimization in the GEOPHIRES-X EGS - Seeking Recommendations

### DISCUSSION THREAD

**Discussion by huhanyu** [NatLabRockies/GEOPHIRES-X#159] [Q&A] [answered]
I am conducting research on optimization methods and hope to use the EGS system in GEOPHIRES-X as a benchmark. I would like to inquire which parameters are most valuable to optimize when tuning the operational parameters of the EGS system? 

I have chosen these parameters **[ProRate, Injection_Temperature, Fracture_Height, Fracture_Separation**] from Example 1 and Example 2 as optimization variables to verify the performance of the optimization method

I don't know if it is appropriate and if there has a better suggestion?

Thank you in advance for your assistance and insights on this matter.

**Comment by softwareengineerprogrammer** [comment]
@malcolm-dsider or @kfbeckers can you help @huhanyu with this question?

**Reply by huhanyu** [reply]
thank you for your help!

**Comment by kfbeckers** [comment, marked as answer]
@huhanyu, what is your objective function? What is it you try to optimize? Maximum electricity production? Minimum LCOE?
Of the parameters you listed, I would say flow rate may be your main operational parameter you can vary. Injection temperature depends on your surface application and you may have limited control over it, especially for electricity production as it will depends on your power plant type and efficiency.
Indeed, you can also vary parameters related to the geometry of the EGS reservoir, which I would qualify under design parameters. These include reservoir depth, number of fractures, fracture height, fracture width (=well spacing), wellbore diameter, number of injection wells, number of production wells.

**Reply by huhanyu** [reply]
I am focused on the economic optimization of EGS, and thus, my objective function aims at minimizing the LOCE/LOCH. I have noticed that the NPV often yields negative values, making it an unsuitable candidate for maximization.
Finally, I would like to express my gratitude for your advice.

**Comment by malcolm-dsider** [comment]
In my experience trying to promote geothermal projects at Shell, to get approval to do the detailed work related to moving a project forward, you had to get general permission from the leadership that spending the money is going to be worth it - economically. So they wanted answers about NPV, IRR, LCOE, etc. first, and then you can get permission to spend the effort to refine your reservoir model, surface plant setup, etc. So you are right to focus on NPV (but also look at IRR, MOIC, etc.) as the primary measure, at least to start. 

But NPV has two parts - the cost to make the product (electricity, in this case), and the profit you will make from selling the product. I think you are missing the second half of the equation, which I think is the BIGGEST lever - the price you will sell the product for. Currently, the default (electricity) pricing model is very conservative - it assumes a constant sale price of 5.5 cents/kWh. If you can't produce your product for less than that number, you will lose money for every KWh you produce, and NPV will be negative. 5.5 cents/kWh is close to the EarthShot LCOE goal, which is difficult to achieve now, especially for first-of-their-kind projects. 

So your biggest levers are "Starting Electricity Sale Price", "Ending Electricity Sale Price", "Electricity Escalation Start Year", and "Electricity Escalation Rate Per Year". If you set "Starting Electricity Sale Price" = "Ending Electricity Sale Price", you will get a constant price - they are both currently set to 0.055 ($/kWh). But you have the option to use "Electricity Escalation Start Year", and "Electricity Escalation Rate Per Year" to set up a non-constant pricing model. "Electricity Escalation Start Year" sets the year that the escalation kicks in (after the start of the project), and "Electricity Escalation Rate Per Year" sets the rate at which the price will grow per year, until it gets to "Ending Electricity Sale Price". The most common setup I use is to set the "Starting Electricity Sale Price" to whatever the local consumer is paying in the areas where you are planning to put the plant (e.g., 0.15 for 15 cents/kWh), "Electricity Escalation Start Year" = 1 (so the rise begins immediately), and the "Electricity Escalation Rate Per Year" to a number that I think will match the long-term inflation rate (something like 2-3%). In this scenario, I set the "Ending Electricity Sale Price" to a very large number because I just want the price to grow continuously with inflation and never stop growing.

**Reply by huhanyu** [reply]
Thank you for sharing your valuable experience. Indeed, during the optimization process, I overlooked the consideration of sales prices. It is true that setting reasonable prices can lead to satisfactory economic indicators. I will continue to experiment in the upcoming trials. Thank you again.
